utils/config/config.go

```diff
@@ -708,6 +709,19 @@ func (serverDetails *ServerDetails) CreateXrayAuthConfig() (auth.ServiceDetails,
 	return serverDetails.createAuthConfig(artAuth)
 }
 
+func (serverDetails *ServerDetails) CreateXscAuthConfig() (auth.ServiceDetails, error) {
+	ascAuth := xscAuth.NewXscDetails()
+	ascAuth.SetUrl(convertXrayUrlToXscUrl(serverDetails.XrayUrl))
```

Comment:
```suggestion
	ascAuth.SetUrl(serverDetails.getXscUrlFromXrayUrl())
```
private method, called once, passing an argument that is also an attribute.
should use different implementation